### PR TITLE
feat: adding ``ansys-mechanical-env`` package

### PR DESCRIPTION
**Pull request by dipinknair**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by koubaa** [comment]
ack

**Comment by RobPasMue** [comment]
I confirm on my side!

**Comment by dipinknair** [comment]
@conda-forge/help-python - ready for review

**Comment by ocefpaf** [comment]
> @conda-forge/help-python - ready for review

It is not passing, so it is not ready. Can you use grayskull to generate that recipe? It will help in the review process.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi

File: recipes/ansys-mechanical-env/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ansys_mechanical_env-{{ version }}.tar.gz
+  sha256: 29a4ce796cf3719828f6af4fa7d1097333cb12d1b10b6ab199efe0f7ec8fc346
+
+build:
```
Comment (RobPasMue) [inline recipes/ansys-mechanical-env/meta.yaml:12]:
```suggestion
build:
  noarch: python
```

File: recipes/ansys-mechanical-env/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  noarch: python
+  skip: true  # [py>=400 or py2k]
```
Comment (RobPasMue) [inline recipes/ansys-mechanical-env/meta.yaml:14]:
```suggestion
  # skip: true  # [py>=400 or py2k]
```

**Comment by RobPasMue** [comment]
@conda-forge/help-python - ready for review

File: recipes/ansys-mechanical-env/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ansys_mechanical_env-{{ version }}.tar.gz
+  sha256: 29a4ce796cf3719828f6af4fa7d1097333cb12d1b10b6ab199efe0f7ec8fc346
+
+build:
+  noarch: python
+  # skip: true  # [py>=400 or py2k]
```
Comment (ocefpaf) [inline recipes/ansys-mechanical-env/meta.yaml:14]:
```suggestion
```

Comment (RobPasMue) [reply recipes/ansys-mechanical-env/meta.yaml]: Yep agreed! Thanks!

**Comment by RobPasMue** [comment]
@ocefpaf - just committed your suggestion. Feel free to approve/review whenever you can =) thanks a lot!

**Pull request merged**